prombench/Makefile

```diff
@@ -68,6 +68,8 @@ funcbench_resource_apply:
 funcbench_resource_delete:
 	$(PROMBENCH_CMD) gke resource delete -a ${AUTH_FILE} \
 		-v ZONE:${ZONE} -v PROJECT_ID:${PROJECT_ID} -v CLUSTER_NAME:${CLUSTER_NAME} \
-		-v PR_NUMBER:${PR_NUMBER} \
+		-v PR_NUMBER:${PR_NUMBER} -v GITHUB_TOKEN:${GITHUB_TOKEN} \
+		-v GITHUB_ORG:${GITHUB_ORG} -v GITHUB_REPO:${GITHUB_REPO} \
+		-v BRANCH:${BRANCH} -v BENCH_FUNC_REGEX:${BENCH_FUNC_REGEX} \
 		-f manifests/funcbench/benchmark/2_job.yaml \
```

Comment:
Is there any order here? i.e `2_job.yaml` needs to be run first and then the namespace for `gke resource delete`? does deleting the nodepool not delete the job?

Reply:
If that's a case will be good to but a small comment next to it about the ordering.^

Reply:
Comment added.